### DISCUSSION TITLE
What model does it use when you choose option 1 in setup.sh?

### DISCUSSION THREAD

**Discussion by naseerfaheem** [arc53/DocsGPT#845] [Q&A] [answered]
I  wanted to see if we could change models once we pick 1. It also has an API_KEY requirement and I couldn't find what that key is? Thanks

**Comment by dartpain** [comment, marked as answer]
It will use docsgpt 7b mistral
[https://huggingface.co/Arc53/docsgpt-7b-mistral](https://huggingface.co/Arc53/docsgpt-7b-mistral)
